src/ContentMessages.js

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 
```

Comment:
You seem to have leaked some of your new upload code in this PR, is this intentional?